sdk/authorization/arm-authorization/package.json

```diff
@@ -2,10 +2,10 @@
   "name": "@azure/arm-authorization",
   "author": "Microsoft Corporation",
   "description": "AuthorizationManagementClient Library with typescript type definitions for node.js and browser.",
-  "version": "8.3.1",
+  "version": "8.3.2",
   "dependencies": {
-    "@azure/ms-rest-azure-js": "^1.1.0",
-    "@azure/ms-rest-js": "^1.1.0",
+    "@azure/ms-rest-azure-js": "^1.3.2",
+    "@azure/ms-rest-js": "^1.8.1",
```

Comment:
update ms-rest-js and ms-rest-azure-js to the latest versions.

Reply:
Why didnt the generated code make this change?

Reply:
Because the generator has not been updated.

Reply:
In that case, the generated code would depend on major version 1 of ms-rest-js. Doesnt forcing it to use the next major version cause breaking changes?

Reply:
ms-rest-nodeauth depends on ^2.0.3 version of ms-rest-js. The changes in ms-rest-azure-js and ms-rest-js are good changes and will work well with the newly generated client. I have tested this with arm-kusto fix that I regenerated last week and everything works well.

The reason for bumping ms-rest-js from 1.x to 2.x was that we moved from axios http client to node-fetch http client for node.js scenario. Made some breaking changes to AbortController api, more info [here](https://github.com/Azure/ms-rest-js/blob/master/Changelog.md#200---2019-06-21). I don't expect many people to be using that feature. So we should be good.

Reply:
@amarzavery Done. Resolving the conversation